### PR TITLE
misc(typings): convert hardcoded strings to networkRequests.TYPES

### PR DESCRIPTION
**Pull request by wardpeet**
**Summary**
Just fixing some changes we missed for resourceType

**Related Issues/PRs**

### CODE REVIEW

**Comment by wardpeet** [comment]
I stumbled upon some other hardcoded strings I'm not sure if they are intentional or not so just checking if I need to change them too.

[estimateTransferSize in the byte-efficiency-audits we call resourceType as a string](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/byte-efficiency/unminified-javascript.js#L59-L60) not sure if we should move to NetworkRecord.TYPES too.

File: lighthouse-core/audits/bootup-time.js [hunk 1]
```diff
@@ -46,7 +47,7 @@ class BootupTime extends Audit {
     /** @type {Set<string>} */
     const urls = new Set();
     for (const record of records) {
-      if (record.resourceType && record.resourceType === 'Script') {
+      if (record.resourceType && record.resourceType === NetworkRequest.TYPES.Script) {
```
Comment (patrickhulce) [inline lighthouse-core/audits/bootup-time.js]: we can simplify this more now too, just `record.resourceType === ` :)

File: lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ class OptimizedImages extends Gatherer {
 
       seenUrls.add(record.url);
       const isOptimizableImage = record.resourceType &&
-        record.resourceType === 'Image' &&
+        record.resourceType === NetworkRequest.TYPES.Image &&
```
Comment (patrickhulce) [inline lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js]: same here :)

**Comment by wardpeet** [comment]
i'll just leave it be and just fix the ones I already have in this PR.

I thank the all seeing eye of @patrickhulce 😄

**Pull request merged**